### PR TITLE
feat(cloudauth): component version

### PR DESCRIPTION
**Pull request by cgeers**

### CODE REVIEW

**Review by haresh-suresh — APPROVED** [review]

File: sysdig/resource_sysdig_secure_cloud_auth_account.go [hunk 1]
```diff
@@ -33,6 +33,10 @@ var (
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			SchemaVersion: {
```
Comment (ravinadhruve10) [inline sysdig/resource_sysdig_secure_cloud_auth_account.go:36]: @cgeers I believe we will need to update the component resource as well with this change.

File: sysdig/resource_sysdig_secure_cloud_auth_account.go [hunk 2]
```diff
@@ -484,6 +490,7 @@ func componentsToResourceData(components []*cloudauth.AccountComponent) []map[st
 		resourceData := map[string]interface{}{}
 		resourceData[SchemaType] = component.GetType().String()
 		resourceData[SchemaInstance] = component.GetInstance()
+		resourceData[SchemaVersion] = component.GetVersion()
```
Comment (ravinadhruve10) [inline sysdig/resource_sysdig_secure_cloud_auth_account.go:493]: Shall we update the ACC tests as well once BE PR is merged?

Comment (cgeers) [reply sysdig/resource_sysdig_secure_cloud_auth_account.go:493]: I don't think this actually depends on new BE functionality. We can add the field in tests now.

Comment (cgeers) [reply sysdig/resource_sysdig_secure_cloud_auth_account.go:493]: done

File: sysdig/resource_sysdig_secure_cloud_auth_account.go (hunk 1 above)
Comment (cgeers) [reply sysdig/resource_sysdig_secure_cloud_auth_account.go:36]: good call. Yes, this already serves the immediate intended purpose, but there's no reason to leave this half implemented. I've added added the conversion steps to the component resourced as well.

**Pull request merged**